service/controller/resource/pvc/local_pvc.go

```diff
@@ -0,0 +1,64 @@
+package pvc
+
+import (
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+)
+
+const (
+	LabelMountTag = "mount-tag"
+)
+
+func newLocalPVCs(customObject v1alpha1.KVMConfig, pvsList *corev1.PersistentVolumeList) []*corev1.PersistentVolumeClaim {
+	var persistentVolumeClaims []*corev1.PersistentVolumeClaim
+	localStorageClass := LocalStorageClass
+
+	for i, workerKVM := range customObject.Spec.KVM.Workers {
+		workerNode := customObject.Spec.Cluster.Workers[i]
+
+		for _, hostVolume := range workerKVM.HostVolumes {
+			for _, pv := range pvsList.Items {
+				if pv.ObjectMeta.Labels[LabelMountTag] == hostVolume.MountTag {
+					persistentVolumeClaim := &corev1.PersistentVolumeClaim{
+						TypeMeta: metav1.TypeMeta{
+							Kind:       "PersistentVolumeClaim",
+							APIVersion: "v1",
+						},
+						ObjectMeta: metav1.ObjectMeta{
+							Name: key.LocalWorkerPVCName(key.ClusterID(customObject), key.VMNumber(i)),
+							Labels: map[string]string{
+								key.LegacyLabelCluster: key.ClusterID(customObject),
+								key.LabelCustomer:      key.ClusterCustomer(customObject),
+								key.LabelApp:           key.WorkerID,
+								key.LabelCluster:       key.ClusterID(customObject),
+								key.LabelOrganization:  key.ClusterCustomer(customObject),
+								key.LabelVersionBundle: key.OperatorVersion(customObject),
```

Comment:
We lost the `node` label here. Did you figure out where to get it?

Reply:
I could see it being useful. This will make the labels match the deployment labels.

Reply:
i put it back. please check it.